lib/renderer.js

```diff
@@ -1164,10 +1183,12 @@
         // 4. The "include-ancestor-masks" config option is NOT set
         // 5. None of the layers has zero bounds. Sometimes they aren't computed and set to all 0's
         // 6. The layer is clipped, in which case layer.bounds is the clipped size and not the layer size
-        var hasComplexTransform = (component.hasOwnProperty("scale") && component.scale % 1 !== 0) ||
-                component.hasOwnProperty("width") || component.hasOwnProperty("height"),
+        var layer = component.layer,
+            hasComplexTransform = layer &&
```

Comment:
Hey @vmitnick or @chrisbank - I believe this restriction (that `hasComplexTransform` can only be true for layer components) is preventing layer comps from using "exact bounds" ... which means that you can not scale layer comps with explicit bounds: Like "200pxx200px Layer Comp.png"

Any memories about this? 

I see the commit msg 
>In _getData(), only be concerned with hasComplexTransform for layers/layer groups, not for whole documents.

Perhaps we should loosen that to "layers, layer groups, or layer comps"?

Is it a reasonable assumption that we don't want to allow explicit height/width for "whole documents"?
cc: @iwehrman